pkg/pillar/docs/zedmanager.md

```diff
@@ -0,0 +1,46 @@
+# Application Instance Management Agent in EVE (aka zedmanager)
+
+## Overview
+
+Zedmanager does the overall orchestration of application instances in EVE.
+This means receiving configuration from the controller via zedagent, and reporting results to the controller via zedagent, and then using other EVE microservices to manage the different parts of the orchestration.
+
+- Using volumemgr to create the (disk) volumes for the application instance
+- Using identitymgr to create any overlay EIDs needed for overlay networking
+- Using zedrouter to create the network adapters and local services like DHCP and DNS for the application instance (note that zedrouter separately sets up the network instances to which these adapters are attached)
+- Using domainmgr to assign any I/O adapters to the application instance
+- Using domainmgr to run the application instance
```

Comment:
domainmgr also stops and deletes the app instance, does it not?

Reply:
I wrote that section on the startup sequence.I added some more sections on delete and modify

Reply:
Ah good point. Makes sense.